providers/marketo/README.md

```diff
@@ -1,11 +1,20 @@
-# Write connnector
+# Marketo Connector
+
+The connector currently supports Leads API only.
 
-Currently the Write connector supports Leads API only. 
+# Write connnector
 
 The objects that we can write to currently are:
```

Comment:
Minor, feel free to do it later, but a table would be better at representing this:

| object | support     |   
|--------|----------|
| lists  | read, write |